.rubocop.yml

```diff
@@ -0,0 +1,41 @@
+Documentation:
+  Enabled: false
+
+AllCops:
+  Exclude:
+    - 'vendor/**/*'
+    - 'tmp/**/*'
+    - 'config/initializers/*'
+    - 'bin/*'
+    - 'db/**/*'
+  TargetRubyVersion: 2.3
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+  EnforcedStyle: always
+
+Style/BlockComments:
+  Description: 'Do not use block comments.'
+  StyleGuide: '#no-block-comments'
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'config/routes.rb'
+    - 'spec/**/*'
+    - 'config/environments/*'
+
+Metrics/MethodLength:
+  Exclude:
+    - 'app/helpers/*'
+
+Metrics/ClassLength:
+  Exclude:
+    - 'app/models/*'
+
+Metrics/LineLength:
+  Max: 120
```

Comment:
isn't that too many?
 90 fair enough

Reply:
There are a lot of comment strings in rails files, that have length between 90 and 100 characters. Can we set Max to 100?